libr/anal/types.c

```diff
@@ -81,11 +81,15 @@ R_API int r_anal_type_get_size(RAnal *anal, const char *type) {
 	}
 	const char *t = sdb_const_get (anal->sdb_types, tmptype, 0);
 	if (!t) {
+		if (!strncmp (tmptype, "enum ", 5)) {
+			//XXX: Need a proper way to determine size of enum
+			return 32;
```

Comment:
yes, but maybe let this to the user, or assume its the same as "int", but that depends on the compiler.. msvc doesnt support 64bit enums :_